packages/ui/src/components/ProgressBar/__tests__/index.test.tsx

```diff
@@ -18,4 +18,40 @@ describe('ProgressBar', () => {
 
   it(`renders correctly when value < 0`, () =>
     shouldMatchEmotionSnapshot(<ProgressBar value={-250} />))
+
+  it(`renders correctly with label, label description and showprogress, direction column`, () =>
```

Comment:
```suggestion
  it(`renders correctly with label, label description and showProgress, direction column`, () =>
```

same for other occurrences in the files